code/modules/mob/living/simple_animal/abnormality/waw/alriune.dm

```diff
@@ -78,7 +78,9 @@
 				if(ishuman(L))
 					var/mob/living/carbon/human/H = L
 					if(H.sanity_lost)
-						H.add_overlay(icon('ModularTegustation/Teguicons/tegu_effects.dmi', "alriune_kill"))
+						new /obj/effect/temp_visual/alriune_curtain(get_turf(H))
```

Comment:
Doesn't Alriune does this effect regardless of kill, it being her "you were hit by my attack" indicator?

Reply:
Also does this fail to check for the target being dead, resulting in potentially infinite overlays stacking on a dead insane person?

Reply:
The death check is above;
The curtains only appear on killed: For normal damage there's generic effect.